controllers/Index.php

```diff
@@ -84,7 +84,7 @@ public function index()
 
         $this->bodyClass = 'compact-container';
         $this->pageTitle = 'rainlab.pages::lang.plugin.name';
-        $this->pageTitleTemplate = '%s Pages';
+        $this->pageTitleTemplate = '%s '.Lang::get('rainlab.pages::lang.plugin.name');
```

Comment:
space between `'` and `.` and `Lang` please

Reply:
Also, why wouldn't this just be a language key straight up and then parsed on render? That way RTL languages would be supported too.